### ISSUE TITLE
Cheat --> wiki

### ISSUE THREAD

**Issue by 7zxkv** [jimgraygit/Onion#152]
hi,

It's not really a problem but just a total misunderstanding and the info is not clear on the net. As with the miyoo we have to do everything by hand (no automatic download), can someone tell me:

- where should we put the cheats for "arcade" (so mame2003+ by default)? what format? (official cheat.7z or oldest cheat.dat or .cht). I tryed here `SD\RetroArch\.retroarch\cheats\MAME 2003-Plus\` and `SD\RetroArch\.retroarch\saves\MAME 2003-Plus\mame2003-plus` it's not good apparently...

- Same question for all regarding FBN or FBA?
The path and the format (fb[a/n] accepts the mame file in its official version, but it also accepts .ini). Moreover how to activate them in the emulator I do not see an option unlike the core of mame.

- There are also standard .cht formats, for whom it belongs and where to put it?

- In short, if you can VERY precisely clarify all this and possibly update the wiki, that would be very much appreciated. Thanks

**Comment by 7zxkv** [comment]
I found for MAME.

For mame (mame2003+ = "arcade" by default)  you must use the old cheat format (.dat and not xml) compatible here:
https://github.com/libretro/mame2003-plus-libretro/blob/master/metadata/cheat.dat

and place here: `SD\BIOS\mame2003-plus`

Activate the mame menu in retroarch option (via menu button)

**Comment by Wapp92** [comment]
Cool thanks alot for the link and explication. 

It would be perfect, if we could configure a button to quick access the MAME menu and change the path of .cht cheats to a permanent folder like bios or roms.

**Comment by Wapp92** [comment]
I put my cheat in the Roms folder change Roms to BIOS, if you prefer your cheat there.

In the retroarch.cfg change the line cheat_database_path to 
cheat_database_path = "/mnt/SDCARD/Roms/cheats"
Or 
cheat_database_path = "/mnt/SDCARD/BIOS/cheats"

Create the cheats folder either in Roms or BIOS. 

Could this be implemented by default in the next build? 

As for configuring the button i have no clue what to change.

**Comment by erazemk** [comment]
> Thanks for your response. Do not hesitate to share links where we can find the cheat codes by emu !

You can find cheats for (most) consoles libretro supports here: https://github.com/libretro/libretro-database/tree/master/cht

**Comment by 7zxkv** [comment]
it is a request that I made here: 
https://github.com/jimgraygit/Onion/issues/152

personally i would agree

**Comment by Sichroteph** [comment]
Leaving this issue open so it can help users in the future.

**Comment by Sichroteph** [comment]
A guide made by Erazemk has been added to the FAQ. Don't hesitate to continue this in the discussion page.
